docs/references/development_workflow.rst

```diff
@@ -82,3 +84,50 @@ Within the learning equality repository:
 If a change needs to be introduced to an older release, target the oldest release branch that we want the change made in. Then that change will need to be merged into all subsequent releases, one-at-a-time, until it eventually gets back to ``develop``.
 
 
+Github labels
+-------------
+
+We use a `wide range of labels <https://github.com/learningequality/kolibri/labels>`__ to help organize issues and pull requests in the Kolibri repo.
+
+
+Priority
+~~~~~~~~
+
+These are used to sort issues and sometimes PRs by priority if *and only if* the item is assigned a milestone. Every issue in a milestone ought to have a priority label.
+
+Only 'critical' items are strictly blockers for a release, but typically all important items should be expected to make it in, too. Priority within a release is generally assigned by a core Learning Equality team member.
+
+* **P0 - critical**
+* **P1 - important**
+* **P2 - normal**
+* **P3 - low**
+
+
+Changelog
+~~~~~~~~~
+
+The **changelog** label is used on PRs or issues to generate 'more details' links in the :ref:`changelog`.
+
+
+Work-in-progress
+~~~~~~~~~~~~~~~~
+
+The **work-in-progress** label is helpful if you have a PR open that's not ready for review yet.
+
+
+Development category
+~~~~~~~~~~~~~~~~~~~~
+
+Labels prefixed with **DEV:** are used to help organize issues (and sometimes PRs) by area of responsibility or scope of domain knowledge necessary.
+
+
+TODO items
+~~~~~~~~~~
+
+Labels prefixed with  **TODO:** help flag items that need some action before the issue or PR can be fully resolved.
+
+
+Organizational Tags
+~~~~~~~~~~~~~~~~~~~
+
+Labels prefixed with **TAG:** are general-purpose, and are used to help organize issues and PRs.
```

Comment:
Rather than sections with one paragraph each, would it be okay to have just a list with bolded items? Or a table?

Reply:
that sounds fine to me but for this one I'll say "PR welcome" :)

Reply:
Right, I'll have a look if it turns out to have the right itch+scratch combo :smile: